testsuite/Main.hs

```diff
@@ -112,22 +112,10 @@ setClashEnvs :: HasCallStack => RunWith -> IO ()
 setClashEnvs Global = setEnv "GHC_ENVIRONMENT" "-"
 setClashEnvs Stack = pure ()
 setClashEnvs Cabal = do
-  -- Make sure environment variable exists
-  let cp = proc "cabal" ["--write-ghc-environment-files=always", "v2-run", "--", "clash", "--help"]
-  (exitCode, stdout, stderr) <- readCreateProcessWithExitCode cp ""
-  case exitCode of
-    ExitSuccess -> do
-      binDir <- cabalClashBinDir
-      path <- getEnv "PATH"
-      setEnv "PATH" (binDir <> ":" <> path)
-      setCabalPackagePaths
-    ExitFailure _ -> do
-      putStrLn "'cabal run clash' failed"
-      putStrLn ">>> stdout:"
-      putStrLn stdout
-      putStrLn ">>> stderr:"
-      putStrLn stderr
-      exitWith exitCode
+  binDir <- cabalClashBinDir
+  path <- getEnv "PATH"
+  setEnv "PATH" (binDir <> ":" <> path)
+  setCabalPackagePaths
```

Comment:
:heavy_check_mark:

Reply:
I really hope this works as expected, because that's one nice diff

Reply:
It does! It _might_ fail on CI as it replaces `cabal.project` I think. We'll see.